tests/regression/56-floats/03-infinity_or_nan.c

```diff
@@ -0,0 +1,11 @@
+// PARAM: --enable ana.float.interval
+#include <stdio.h>
+
+void main()
+{
+    double data;
+    fscanf(stdin, "%lf", &data);
+    
+    double result1 = data + 1.0; // WARN: Could be +/-infinity or Nan
```

Comment:
what about some additional examples inside more elaborate constructs
```c
if ((int)(data - 1.)) // WARN: Could be +/-infinity or Nan
    return 0; // `if` requires an actual body, otherwise it is ignored
while ((int)(data - 2.)); // WARN: Could be +/-infinity or Nan
assert ((int)(data - 3.)); // WARN: Could be +/-infinity or Nan
```

Reply:
Do they actually give any more information, besides maybe that it's also warning for minus? I personally prefer it to keep it as simple as needed, and in this case we are not checking for any control flow. But I can also change it if you think it's better :)